### DISCUSSION TITLE
1SWEAY23/24_02230289_KUENZANG_RABTEN

### DISCUSSION THREAD

**Discussion by Rabtens** [kevinlin1/just-the-class#40] [Q&A]
https://www.figma.com/file/TBkOUzpvJfJnM5Pnsxwxzh/Untitled?type=design&node-id=0-1&mode=design&t=MDfjrI2dHZ9KFrcm-0

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
